csharp/Platform.Collections/Segments/Walkers/AllSegmentsWalkerBase[T, TSegment].cs

```diff
@@ -6,41 +6,34 @@
 namespace Platform.Collections.Segments.Walkers
 {
     /// <summary>
-    /// <para>
-    /// Represents the all segments walker base.
-    /// </para>
-    /// <para></para>
+    /// <para>Represents the base class for walkers through all possible segments of a sequence.</para>
+    /// <para>Представляет базовый класс для проходчиков по всем возможным сегментам последовательности.</para>
     /// </summary>
     /// <seealso cref="AllSegmentsWalkerBase"/>
+    /// <seealso cref="AllSegmentsWalkerBase{T}"/>
     public abstract class AllSegmentsWalkerBase<T, TSegment> : AllSegmentsWalkerBase
         where TSegment : Segment<T>
     {
         /// <summary>
-        /// <para>
-        /// The minimum string segment length.
-        /// </para>
-        /// <para></para>
+        /// <para>A read-only field that represents the minimum length of the sequence segment.</para>
+        /// <para>Поле только для чтения, которое представляет минимальную длину сегмента последовательности.</para>
         /// </summary>
         private readonly int _minimumStringSegmentLength;
 
         /// <summary>
-        /// <para>
-        /// Initializes a new <see cref="AllSegmentsWalkerBase"/> instance.
-        /// </para>
-        /// <para></para>
+        /// <para>Constructor that initializes the field <see cref="_minimumStringSegmentLength"/>, inserts the default value there.</para>
+        /// <para>Конструктор, который инициализирует поле <see cref="_minimumStringSegmentLength"/> , вставляет туда значение по умолчанию.</para>
         /// </summary>
         /// <param name="minimumStringSegmentLength">
-        /// <para>A minimum string segment length.</para>
-        /// <para></para>
+        /// <para>The parameter that takes a field <see cref="DefaultMinimumStringSegmentLength"/>.</para>
```

Comment:
This description is not descriptive. [Check how microsoft documentate parameters](https://docs.microsoft.com/en-us/dotnet/api/system.io.filestream.-ctor?view=net-5.0#System_IO_FileStream__ctor_Microsoft_Win32_SafeHandles_SafeFileHandle_System_IO_FileAccess_)